stable/hlf-ca/tests/README.md

```diff
@@ -0,0 +1,75 @@
+# Chart testing
+
+> Eventually this will be replaced with an integration test, likely running with `pytest`
+
+Commands should be run from the root folder of the repository.
+
+## Root CA
+
+### Install
+
+Install Root CA
+
+    helm install ./hlf-ca -n rca --namespace test -f ./hlf-ca/tests/values/root.yaml
```

Comment:
Running from `root folder` can't find it:
```
╰─➤  helm install ./hlf-ca -n rca --namespace test -f ./hlf-ca/tests/values/root.yaml
Error: path "./hlf-ca" not found
```
Probably you need to add `/stable` before actual path. 

Running from `stable` I get this:
```╰─➤  helm install ./hlf-ca -n rca --namespace test -f ./hlf-ca/tests/values/root.yaml
Error: found in requirements.yaml, but missing in charts/ directory: postgresql, mysql
```

Reply:
Aha, I forgot to add:
```
helm dependency update
```

I will add